client/src/client/system_info/web.rs

```diff
@@ -3,6 +3,7 @@ pub(crate) fn get_num_cores() -> usize {
     js_result.as_f64().unwrap_or(1.0) as usize
 }
 
+#[cfg(feature = "desktop")]
```

Comment:
This file is already specifically for web. We could remove this function but it might be useful later. Could just do a clippy ignore or something.